tests/php/test-class-amp-tiktok-embed-handler.php

```diff
@@ -92,10 +136,15 @@ public function get_conversion_data() {
 	 * @param string $expected Expected.
 	 * @dataProvider get_conversion_data
 	 */
-	public function test_conversion( $source, $expected ) {
+	public function test_conversion( $source, $expected = null ) {
 		if ( version_compare( '5.4-alpha', get_bloginfo( 'version' ), '>' ) ) {
 			$this->markTestSkipped( 'The TikTok embed is only available in 5.4-alpha (until 5.4 is stable)' );
 		}
```

Comment:
This can now be removed now that the TikTok embed is generally available in WP.

Reply:
It's not available in older versions of WP still, right?

Reply:
Support for TikTok was added in WP 5.4: https://core.trac.wordpress.org/ticket/49083

So in versions older than 5.4 we wouldn't want to test this.

Reply:
Ah right, we could update the message then.